glsk/glsk-document-io-api/src/main/java/com/powsybl/glsk/api/io/GlskDocumentImporter.java

```diff
@@ -26,5 +26,9 @@ default GlskDocument importGlsk(InputStream inputStream, boolean useCalculationD
         throw new NotImplementedException("This importer does not handle CalculationDirections");
     }
 
+    default GlskDocument importAndValidateGlsk(InputStream inputStream, boolean useCalculationDirections) {
```

Comment:
You probably should add a new method in the `GlskDocumentImporters` class to access this new `importAndValidateGlsk()` method, as I did for `importGlskWithCalculationDirections()` method.

Reply:
Sorry I forgot that one, I will even add two methods ;) one for validate and on for validate with calculation direction